packages/core/scripts/collateral-umip/1_Propose.js

```diff
@@ -46,34 +56,43 @@ async function runExport() {
 
   const collaterals = _.castArray(argv.collateral);
   const fees = _.castArray(argv.fee);
-  const decimals = argv.decimals && _.castArray(argv.decimals);
```

Comment:
We don't need decimals to be passed in since we try to auto detect it via `getDecimals` which works for any ERC20 that implements `decimals()` which should be all.